src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
     }
```

Comment:
Since `toString()` can be overridden and its behavior changed, it's recommended to use `name()` for getting the String representation of enum constants.